src/instance.ts

```diff
@@ -1084,6 +1091,58 @@ Please use the format 'my-instance' or '${bigtable.projectName}/instances/my-ins
     ]);
   }
 
+  createTableFromBackup(
+    config: CreateTableFromBackupConfig
```

Comment:
Question about passing `tableId` separately from the rest of the config to have a similar feel as `Instance.createTable()`